internal/gitutil/gitutil.go

```diff
@@ -32,10 +32,34 @@ import (
 // for remote repos.  Defaults to UserHomeDir/.kpt/repos if unspecified.
 const RepoCacheDirEnv = "KPT_CACHE_DIR"
 
+var DefaultRef = "main"
```

Comment:
We should try to avoid these global variables.

Reply:
Done.